website/users.php

```diff
@@ -289,7 +289,8 @@
 				Use this link:
 			</p>
 			<p style="margin: 1em -13px">
-				<small><code>https://<?= $psconfig['routes']['root'] ?>/resetpassword/<?php echo $token; ?></code></small>
+				<small><code id="token">https://<?= $psconfig['routes']['root'] ?>/resetpassword/<?php echo $token; ?></code></small>
```

Comment:
IDs are required to be unique by the spec; a static ID can't be unique if it's in a panel.

The best way to do this is to use a class and use a local search. You also want to replace the `onclick` handler with a Backbone click handler in `ladder.js`

Reply:
So find this:

```js
	events: {
		'change select[name=standing]': 'changeStanding',
		'click button[name=openReset]': 'openReset',
		'click button[name=cancelReset]': 'cancelReset'
	},
```

and add your copy button:

```js
	events: {
		'change select[name=standing]': 'changeStanding',
		'click button[name=copyLink]': 'copyLink',
		'click button[name=openReset]': 'openReset',
		'click button[name=cancelReset]': 'cancelReset'
	},
```

Reply:
Then you can follow along how `openReset` works.

Reply:
Okay. Done.